electron/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "unified-auctions",
+  "version": "1.0.0",
+  "description": "MakerDAO Unified Auctions Electron App",
+  "main": "main.js",
+  "build": {
+    "appId": "com.makerdao.unified-auctions",
+    "mac": {
+      "category": "public.app-category.finance"
+    },
+    "directories": {
+      "output": "out"
+    },
+    "files": [
+      "main.js",
+      "dist/**/*"
+    ]
+  },
+  "scripts": {
+    "preinstall": "cd ../frontend && npm ci && npm run build && npm run generate",
+    "postinstall": "electron-builder install-app-deps",
+    "prepare": "npx copyfiles -a -u 2 \"../frontend/dist/**/*\" .",
```

Comment:
When is this command actually executed?

Reply:
You should not normally mix installation with running/building commands. Otherwise, why is it not a default behaviour? 🙂. Expected flow: developer cds into electron folder run `npm run install` only once to install every required dependency. Then, they should be able to run `npm run build` which will rebuild the frontend and build a new app